adyenv6b2ccheckoutaddonv2/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen-checkout/src/components/payment/Payment.tsx

```diff
@@ -131,6 +165,22 @@ class Payment extends React.Component<Props, State> {
         dropIn.mount(this.paymentRef.current)
     }
 
+    private prepareAdyenPaymentForm(): AdyenPaymentForm {
```

Comment:
it was moved to payment service - replaced with form for bank card and stored card